tests/recovery.py

```diff
@@ -113,16 +113,18 @@ def run(args):
             args.perf_nodes,
             node_offset=(recovery_idx + 1) * len(hosts),
             pdb=args.pdb,
-        ) as network:
-            primary, backups = network.start_in_recovery(args, ledger, sealed_secrets)
+        ) as recovered_network:
+            primary, backups = recovered_network.start_in_recovery(
+                args, ledger, sealed_secrets
+            )
 
             with primary.management_client() as mc:
                 check_commit = infra.ccf.Checker(mc)
                 check = infra.ccf.Checker()
 
-                for node in network.nodes:
+                for node in recovered_network.nodes:
                     wait_for_state(node, b"partOfPublicNetwork")
-                network.wait_for_node_commit_sync()
+                recovered_network.wait_for_node_commit_sync()
                 LOG.success("Public CFTR started")
 
                 LOG.debug(
```

Comment:
This would never be called as `network.get_members()` always returned an empty list. Why is that? Because the recovered network was also named `network` and the recovered network does not create new members (rightly so).